lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:test_url)
```

Comment:
If I remove the defaults from `litle.yml` and make this change, then I get the same error as Travis. The reason is that the `commit` method in `litle.rb` accesses the `test_url` and passes it to `ssl_post`, but the url will be nil:
```
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/connection.rb:67:in `request'
/Users/adudzinskaya/.rvm/rubies/jruby-1.7.22/lib/ruby/1.9/benchmark.rb:295:in `realtime'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/connection.rb:60:in `request'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/network_connection_retries.rb:23:in `retry_exceptions'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/network_connection_retries.rb:46:in `retry_network_exceptions'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/network_connection_retries.rb:21:in `retry_exceptions'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/connection.rb:54:in `request'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/posts_data.rb:65:in `raw_ssl_request'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/posts_data.rb:39:in `ssl_request'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/posts_data.rb:35:in `ssl_post'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/billing/gateways/litle.rb:294:in `commit'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/activemerchant-1.55.0/lib/active_merchant/billing/gateways/litle.rb:120:in `store'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/killbill-8.3.0/lib/killbill/helpers/active_merchant/gateway.rb:71:in `method_missing'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/killbill-8.3.0/lib/killbill/helpers/active_merchant/gateway.rb:28:in `store'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/killbill-8.3.0/lib/killbill/helpers/active_merchant/payment_plugin.rb:160:in `add_payment_method'
/Users/adudzinskaya/projects/killbill-litle-plugin/lib/litle/api.rb:119:in `add_payment_method'
/Users/adudzinskaya/.rvm/gems/jruby-1.7.22/gems/killbill-8.3.0/lib/killbill/helpers/active_merchant/killbill_spec_helper.rb:28:in `create_payment_method'
``